content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown

```diff
@@ -0,0 +1,72 @@
+categories:
+  - bytesized
+  - R
+tags:
+  - rmarkdown
+  - knitr
+toc: no
+images: ~
+---
+
+```{r, include=FALSE}
+Sys.setenv(lang = "en_US")
+knitr::opts_chunk$set(collapse = TRUE)
+```
+
+<!-- `Rmarkdown` is a great tool for creating data-driven reports that have to be updated on a regular basis. To communicate to stakeholders that the report is indeed up to date it's important to display the current date below the title. -->
+
+`Rmarkdown` es una herramienta fantástica para crear informes impulsados por datos que requieran actualizarse regularmente. Para que los lectores sepan que el informe está realmente actualizado, es importante mostrar la fecha actual debajo del título.
+
+<!-- An obvious solution to display the current date is to update the date field in the YAML header manually before knitting. But this is both error prone and you are likely to forget about it. Instead you can dislay the current date in an automatic way using the method below. -->
+
+Una solución obvia para mostrar la fecha actual sería modificar manualmente el campo de la fecha en el encabezado YAML antes de compilar el informe (**knit**). Pero esto puede conllevar errores y es probable que se te olvide actualizarlo. En cambio, puedes mostrar la fecha actualizada automáticamente usando el método a continuación.
+
+```{r, eval=FALSE}
+---
+title: "Super Important Report"
+author: "Thomas Neitmann"
+date: `r Sys.Date()`
+---
+```
+
+<!-- When knitting the `Rmarkdown` document `Sys.Date()` will be evaluated and display the current date. -->
+
+Cuando se compila (**knit**) el documento `Rmarkdown`, la función `Sys.Date()` será evaluada y mostrará la fecha actual.
+
+```{r}
+Sys.Date()
+```
+
+<!-- A caveat is the date format, though. `Sys.Date()` returns the current date in ISO 8601 format, i.e. `yyyy-mm-dd`. Your audience is likely not used to this format so you might want to use a date format that's common in your locale. -->
+
+<!-- How to achieve that? Using the `format()` function. Here are some examples. -->
+
+Sin embargo, hay que tener cuidado con el formato de la fecha. `Sys.Date()` devuelve la fecha actual con el formato ISO 8601 (`yyyy-mm-dd`). Es probable que tus lectores no estén acostumbrados a este formato y, por lo tanto, quieras utilizar el formato que se usa en tu región.
+
+¿Cómo podemos hacerlo? Usando la función `format()`. He aquí algunos ejemplos.
+
+```{r}
+current_date <- Sys.Date()
+format(current_date, "%d.%m.%Y")
+format(current_date, "%d. %B %Y")
+format(current_date, "%m/%d/%Y")
+```
+
+<!-- After deciding which format to use make sure to wrap `Sys.Date()` inside of `format()` in your YAML header. -->
+
+Una vez que hayas decidido qué formato utilizar, incluye `Sys.Date()` dentro de `format()` en el encabezado YAML.
+
+```{r, eval=FALSE}
+---
+title: "Super Important Report"
+author: "Thomas Neitmann"
+date: `r format(Sys.Date(), "%d. %B %Y")`
+---
+```
+
+Este artículo ha sido traducido del original en inglés por Gustavo Zapata Wainberg.
```

Comment:
Would you like to include a link to your Twitter or elsewhere?

Reply:
I think that it is enough with just the name